.test/config-giab/config.yaml

```
@@ -50,6 +50,9 @@ mutational_burden:
     - somatic_tumor_low
 
 # printing of variants in a matrix, sorted by recurrence
+maftools:
```

```suggestion
maf:
```
Suggestion: we might switch to a different tool in the future, but the output is MAF format, so this way we make the config independent of the tool. Also, I would name the output folders accordingly, i.e. `results/maf` instead of `results/maftools`.

---

Good idea, done in: https://github.com/snakemake-workflows/dna-seq-varlociraptor/pull/297/commits/d38cced638c7e19875eef4ba0856b7d23950ed43